data_analysis/pii/analysis.ipynb

```diff
@@ -562,6 +568,398 @@
+      "('a9def2a2-54be-40d4-83bf-efc34cc2fbbc', (1924, 1961), 'regex.Regex(\\'((?:(?:[A-Za-z]+[\\\\\\\\p{Nd}\\\\\\\\p{Pd}\\\\\\\\/\\\\\\\\+\\\\\\\\=:_]+|[\\\\\\\\p{Nd}\\\\\\\\p{Pd}\\\\\\\\/\\\\\\\\+\\\\\\\\=:]+[A-Za-z]+)){10,})(?:$|[\\\\\\\\b\\\\\\\\s\\\\\\\\p{Han}@?,!;:\\\\\\\\\\\\\\'\\\\\\\\\")(.])\\', flags=regex.M | regex.V0)', 'KEY')\n",
+      "\n",
+      "===============152==================\n",
+      "('file:////Users/tony/Dropbox/Projects/UW/cse599c-17sp-projects/spark-advantage/data/', (171, 255), 'regex.Regex(\\'((?:(?:[A-Za-z]+[\\\\\\\\p{Nd}\\\\\\\\p{Pd}\\\\\\\\/\\\\\\\\+\\\\\\\\=:_]+|[\\\\\\\\p{Nd}\\\\\\\\p{Pd}\\\\\\\\/\\\\\\\\+\\\\\\\\=:]+[A-Za-z]+)){10,})(?:$|[\\\\\\\\b\\\\\\\\s\\\\\\\\p{Han}@?,!;:\\\\\\\\\\\\\\'\\\\\\\\\")(.])\\', flags=regex.M | regex.V0)', 'KEY')\n",
+      "\n",
+      "===============154==================\n",
+      "('org/wiki/Proposals_for_closing_projects/Closure_of_Simple_English_Wikibooks_', (4078, 4155), 'regex.Regex(\\'((?:(?:[A-Za-z]+[\\\\\\\\p{Nd}\\\\\\\\p{Pd}\\\\\\\\/\\\\\\\\+\\\\\\\\=:_]+|[\\\\\\\\p{Nd}\\\\\\\\p{Pd}\\\\\\\\/\\\\\\\\+\\\\\\\\=:]+[A-Za-z]+)){10,})(?:$|[\\\\\\\\b\\\\\\\\s\\\\\\\\p{Han}@?,!;:\\\\\\\\\\\\\\'\\\\\\\\\")(.])\\', flags=regex.M | regex.V0)', 'KEY')\n",
+      "\n",
+      "===============154==================\n",
+      "('org/wiki/Proposals_for_closing_projects/Closure_of_Basa_Sunda_Wikibooks_', (4209, 4282), 'regex.Regex(\\'((?:(?:[A-Za-z]+[\\\\\\\\p{Nd}\\\\\\\\p{Pd}\\\\\\\\/\\\\\\\\+\\\\\\\\=:_]+|[\\\\\\\\p{Nd}\\\\\\\\p{Pd}\\\\\\\\/\\\\\\\\+\\\\\\\\=:]+[A-Za-z]+)){10,})(?:$|[\\\\\\\\b\\\\\\\\s\\\\\\\\p{Han}@?,!;:\\\\\\\\\\\\\\'\\\\\\\\\")(.])\\', flags=regex.M | regex.V0)', 'KEY')\n",
+      "\n",
+      "===============159==================\n",
+      "('com/torvalds/linux/blob/2bfe01ef/include/uapi/linux/binfmts', (5569, 5629), 'regex.Regex(\\'((?:(?:[A-Za-z]+[\\\\\\\\p{Nd}\\\\\\\\p{Pd}\\\\\\\\/\\\\\\\\+\\\\\\\\=:_]+|[\\\\\\\\p{Nd}\\\\\\\\p{Pd}\\\\\\\\/\\\\\\\\+\\\\\\\\=:]+[A-Za-z]+)){10,})(?:$|[\\\\\\\\b\\\\\\\\s\\\\\\\\p{Han}@?,!;:\\\\\\\\\\\\\\'\\\\\\\\\")(.])\\', flags=regex.M | regex.V0)', 'KEY')\n",
+      "\n",
+      "===============195==================\n",
+      "('372f0f95-6b48-487a-a1ac-383ba580fc1c', (1951, 1988), 'regex.Regex(\\'((?:(?:[A-Za-z]+[\\\\\\\\p{Nd}\\\\\\\\p{Pd}\\\\\\\\/\\\\\\\\+\\\\\\\\=:_]+|[\\\\\\\\p{Nd}\\\\\\\\p{Pd}\\\\\\\\/\\\\\\\\+\\\\\\\\=:]+[A-Za-z]+)){10,})(?:$|[\\\\\\\\b\\\\\\\\s\\\\\\\\p{Han}@?,!;:\\\\\\\\\\\\\\'\\\\\\\\\")(.])\\', flags=regex.M | regex.V0)', 'KEY')\n",
+      "\n",
+      "===============195==================\n",
+      "('372f0f95-6b48-487a-a1ac-383ba580fc1c', (3837, 3874), 'regex.Regex(\\'((?:(?:[A-Za-z]+[\\\\\\\\p{Nd}\\\\\\\\p{Pd}\\\\\\\\/\\\\\\\\+\\\\\\\\=:_]+|[\\\\\\\\p{Nd}\\\\\\\\p{Pd}\\\\\\\\/\\\\\\\\+\\\\\\\\=:]+[A-Za-z]+)){10,})(?:$|[\\\\\\\\b\\\\\\\\s\\\\\\\\p{Han}@?,!;:\\\\\\\\\\\\\\'\\\\\\\\\")(.])\\', flags=regex.M | regex.V0)', 'KEY')\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
```

Comment:
could you add a title or a comment re. what this snippet is doing?

Reply:
Sorry for that. It is still work in progress. I share the PR so if others want to run some experiment they can do it. I will still do some modifications and improve the code quality